guide/additional-info/hosting.md

```diff
@@ -0,0 +1,68 @@
+# Hosting your bot on a VPS
+
+A bot requires itself to be hosted in order for it to run, however, it would not be ideal to host it on your computer all day without shutting it down. That's where a **Virtual Private Server** (VPS) comes into place. A good VPS can mark a turning point for your project as you can host local databases, and it is reliable as well. A VPS suitable for your Discord bot is typically cheap, ranging from around $2-10/month.
+
+## Well known affordable providers
+
+We have gathered a small list of reputable and inexpensive VPS providers that you may rely on for getting your job done:
+* [OVH](https://www.ovh.com/us/vps/) - Starting at $3.35/month for 1 core, 2GB RAM, 20GB SSD
+* [DigitalOcean](https://www.digitalocean.com/) - Starting at $5/month for 1 core, 1GB RAM, 25GB SSD
+* [Linode](https://www.linode.com/) - Starting at $5/month for 1 core, 1GB RAM, 25GB SSD
+* [Vultr](https://www.vultr.com/) - Starting at $2.50/month for 1 core, 512MB RAM, 10GB SSD
+* [AWS Lightsail](https://amazonlightsail.com/) - Starting at $3.50/month (first month free) for 1 core, 512MB RAM, 20GB SSD
+
+Before you purchase a VPS, we recommend learning how to remotely connect to a VPS. The usual method is via [SSH](https://en.wikipedia.org/wiki/Secure_Shell_Protocol). We won't go deep into it as it's outside the scope of this guide, however, VPS providers usually have a guide regarding connection so you can read up on that.
+
+## Steps to host
+
+::: warning
+The following steps stated assumes your Linux distribution is Ubuntu. However, all Linux distributions are similar and should work on your VPS.
```

Comment:
```suggestion
The following steps assume your VPS is using the Ubuntu Linux distribution. Most hosting providers will offer Ubuntu as an option; however, if your VPS doesn't have Ubuntu, you may need to change a few of the steps below (such as `apt-get`).
```

Reply:
Good idea, I'll keep it on hold for now, most likely will commit soon though

Reply:
Why keep it on hold?